### DISCUSSION TITLE
First timers test discussion

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#424] [First-Timers]
We can start discussions for people to create issues to get a green square for today

**Comment by hasthamalp** [comment]
what i start

**Reply by eddiejaoude** [reply]
I recommend start by improving your GitHub profile, it will look more professional:

- Adding a GitHub profile picture 
- Adding a GitHub profile description 

This was just a test discussion, so glad you are joining in the conversation and practicing with us 🤓 . Have a look at other discussions with the same topic of `First-Timers` and see what looks interesting to you, you can then raise an Issue for a green square and see if you would like to raise a Pull Request too

**Reply by hasthamalp** [reply]
thank you
